wrappers/wingdi.go

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) 2014-2017 MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the license is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package wrappers
+
+type DISPLAY_DEVICE struct {
+	Cb           uint32
+	DeviceName   [32]uint16
+	DeviceString [128]uint16
+	StateFlags   uint32
+	DeviceID     [128]uint16
+	DeviceKey    [128]uint16
+}
+
+const (
+	DISPLAY_DEVICE_ACTIVE           = 0x00000001
+	DISPLAY_DEVICE_ATTACHED         = 0x00000002

```

Comment:
It appears that `DISPLAY_DEVICE_ATTACHED` is undocumented and should therefore be omitted.  (If you've found documentation for it somewhere that I missed, please let me know.)

Reply:
Indeed is undocumented, so I'm removing it.